src/Backend/SgateShopgatePlugin/Components/User.php

```diff
@@ -123,15 +136,6 @@ public function loginUser($request, $httpResponse)
             $httpResponse->setHttpResponseCode(401);
             $httpResponse->setBody(json_encode($error));
         } else {
-            if (!empty($basket['content'])) {
-                $this->basket->clearBasket();
-                $this->basket->sRefreshBasket();
-
-                foreach ($basket['content'] as $basketItem) {
-                    $this->basket->sAddArticle($basketItem['ordernumber'], $basketItem['quantity']);
-                }
-            }
-
```

Comment:
Once the cart-loss bug was fixed, this workaround for restoring its contents could be removed